.github/workflows/concrete_python_tests_linux.yml

```diff
@@ -72,147 +75,140 @@ jobs:
           options: >-
             -v ${{ github.workspace }}:/concrete
             -v ${{ github.workspace }}/build:/build
-            -v ${{ env.SSH_AUTH_SOCK }}:/ssh.socket
-            -e SSH_AUTH_SOCK=/ssh.socket
           shell: bash
           run: |
-            rustup toolchain install nightly-2024-09-30
-            pip install mypy
             set -e
-            rm -rf /build/*
 
+            rustup toolchain install nightly-2024-09-30
             dnf -y install graphviz graphviz-devel
 
+            #rm -rf /build/*
+
             cd /concrete/frontends/concrete-python
             make venv
             source .venv/bin/activate
-            
+
             cd /concrete/compilers/concrete-compiler/compiler
             make BUILD_DIR=/build DATAFLOW_EXECUTION_ENABLED=ON CCACHE=ON Python3_EXECUTABLE=$(which python3) python-bindings
-            
+
             echo "Debug: ccache statistics (after the build):"
             ccache -s
-
-      - name: Prepare test environment
-        uses: addnab/docker-run-action@4f65fabd2431ebc8d299f8e5a018d79a769ae185 # v3
-        if: ${{ !contains(inputs.instance_type, 'p3') }}
+      - name: Slack Notification
```

Comment:
Put the Slack notification after the upload to be notified if the upload failed.